Feature: New links and update get-involved contact & homepage.md (#495)
cds-snc/gcds-docs: src/en/get-involved.md

A design system is never complete. We're committed to building with the people who use our products.

Right now, we're testing GC Design System in alpha, the first usable phase of a product. We’re iterating based on common needs across the GC and those of the people we all serve.

## Find out about GC Design System

> <gcds-grid class="mb-400" columns-tablet="1fr 1fr">

Author: ```suggestion
<gcds-grid class="mb-400 hydrated" columns-tablet="1fr 1fr 1fr">
```

> <gcds-card
>   card-title="Subscribe to newsletter"
>   href="#https://design-system.alpha.canada.ca/en/contact/"
>   badge="badge"
>   description="Subscribe to our newsletter to get GC Design System updates, release communications, and special events. "
> >
> <gcds-card
>   card-title="Attend a demo"
>   href="#https://design-system.alpha.canada.ca/en/register-for-a-demo/" 
>   badge="badge"
>   description="Get an intro to prototyping and developing web experiences using the design system, followed by a Q&A. "
> >

Reviewer: This change will make the cards be side by side, fixes the render of the card, removes the badge and fixes the links.

```suggestion
<gcds-grid class="mb-400" columns-tablet="1fr 1fr">
  <gcds-card
    card-title="Subscribe to newsletter"
    href="{{ links.contact }}"
    description="Subscribe to our newsletter to get GC Design System updates, release communications, and special events."
  ></gcds-card>
  <gcds-card
    card-title="Attend a demo"
    href="{{ links.registerDemo }}"
    description="Get an intro to prototyping and developing web experiences using the design system, followed by a Q&A."
  ></gcds-card>
</gcds-grid>
```

- **Code**: prototype or production code, or product implementations.

<gcds-link href="{{ links.githubCompsPriority }}" external>Contribute to next priorities</gcds-link>
Provide any of the following for each component or pattern:
- **Examples of designs**: screenshots, prototypes, links to live services.
- **Research findings**: your discovery, use cases, or usability testing
- **Code**: prototype or production code, or product implementations.

> <gcds-grid class="mb-400 hydrated" columns-tablet="1fr 1fr">
> <gcds-button href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
> <gcds-button href="{{ links."/fr/contactez" }}" external>Contactez-nous</gcds-button>
> 
> </gcds-grid>

Reviewer: - Buttons should not need a grid component to render side by side.
- Also there is a typo in the links data that is stopping the site from building. 
- Changed buttons to secondary buttons since we shouldn't have two primary buttons on the same page.
- Link text is in French

```suggestion
<gcds-button button-role="secondary" button-type="link" href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
<gcds-button button-role="secondary" button-type="link" href="{{ links.contact }}" external>Contactez-nous</gcds-button>
```

<gcds-link href="{{ links.githubCompsPriority }}" external>Contribute to next priorities</gcds-link>

Provide any of the following for each component or pattern:

- **Examples of designs**: screenshots, prototypes, links to live services.
- **Research findings**: your discovery, use cases, or usability testing
- **Code**: prototype or production code, or product implementations.

<gcds-button button-role="secondary" button-type="link" href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>

> <gcds-button button-role="secondary" button-type="link" href="{{ links.contact }}" external>Contactez-nous</gcds-button>

Reviewer: Will need to put the English text in these links

> 

Author: ```suggestion
Provide any of the following for each component or pattern:
- **Examples of designs**: screenshots, prototypes, links to live services.
- **Research findings**: your discovery, use cases, or usability testing
- **Code**: prototype or production code, or product implementations.

<gcds-grid class="mb-400 hydrated" columns-tablet="1fr 1fr">
<gcds-button href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
<gcds-button href="{{ links."/fr/contactez" }}" external>Contactez-nous</gcds-button>

</gcds-grid>
```

<hr />

<gcds-heading tag="h2" margin-top="225">What else is coming soon</gcds-heading>

### Upcoming components and templates

Here’s what we’ll soon be releasing. More on our roadmap.

<div>

>   <gcds-heading tag="h4" margin-bottom="0">Header V2</gcds-heading>

Reviewer: Are we able to label these something different then `V2`? I feel like that is an internal designation and won't mean much to people reading our docs site since we don't version components individually.

Author: Yeah we could use update instead

>   <gcds-heading tag="h4" margin-bottom="0">Header V2</gcds-heading>

Author: ```suggestion
  <gcds-heading tag="h4" margin-bottom="0">Header update</gcds-heading>
```

  <ul class="mb-300">
    <li>We’re making updates to the header to match the Canada.ca global header pattern.</li>
    <li><strong>Expected release:</strong> Spring 2025</li>
  </ul>
</div>

<div>

>   <gcds-heading tag="h4" margin-bottom="0">Footer V2</gcds-heading>

Author: ```suggestion
  <gcds-heading tag="h4" margin-bottom="0">Footer update</gcds-heading>
```

  <ul class="mb-300">

>     <li>We’re making updates to the footer to match the Canada.ca global header pattern.</li>

Reviewer: ```suggestion
    <li>We’re making updates to the footer to match the Canada.ca global footer pattern.</li>
```

    <li><strong>Expected release:</strong> Spring 2025</li>
  </ul>
</div>
<div>

>   <gcds-heading tag="h4" margin-bottom="0">Button V2</gcds-heading>

Author: ```suggestion
  <gcds-heading tag="h4" margin-bottom="0">Button update</gcds-heading>
```
